pkg/resources/marketplace/manifestHelper.go

```diff
@@ -66,10 +66,9 @@ func GetFilesFromManifestAsStringList(dir string, regex string, packageYaml stri
 			}
 
 			return err
-		} else { // Otherwise find all matching files and process to a string list
-			if err == nil && libRegEx.MatchString(info.Name()) {
-				return ProcessYamlFile(path, &stringList)
-			}
+		} // Otherwise find all matching files and process to a string list
```

Comment:
Now that this doesnt have an `else`, I think the comment would be better placed on a line by itself:

```suggestion
		}
		// Otherwise find all matching files and process to a string list
```